examples/image_classifier/resnet_152_batch/README.md

```diff
@@ -3,7 +3,7 @@ Run the commands given in following steps from the parent directory of the root
 
 ```bash
 wget https://download.pytorch.org/models/resnet152-b121ed2d.pth
-torch-model-archiver --model-name resnet-152-batch --version 1.0 --model-file ./serve/examples/image_classifier/resnet_152_batch/model.py --serialized-file resnet152-b121ed2d.pth --handler ./serve/examples/image_classifier/resnet_152_batch/resnet152_handler.py --extra-files ./serve/examples/image_classifier/index_to_name.json
+torch-model-archiver --model-name resnet-152-batch --version 1.0 --model-file ./serve/examples/image_classifier/resnet_152_batch/model.py --serialized-file resnet152-b121ed2d.pth --handler image_classifier --extra-files ./serve/examples/image_classifier/index_to_name.json
```

Comment:
@shivamshriwas Is the --handler value correct? Shouldn't it be the actual file path for the handler .py file

Reply:
@chauhang  resnet152_handler has been removed due to batch support in default handlers. Hence the above change